hedera-node/hedera-app/src/test/java/com/hedera/node/app/state/merkle/MerkleTestBase.java

```diff
@@ -204,33 +229,41 @@ protected void setupConstructableRegistry() {
         }
     }
 
-    /** Creates a new arbitrary merkle map with the given label. */
+    /**
+     * Creates a new arbitrary merkle map with the given label.
+     */
     protected <K extends Comparable<K>, V> MerkleMap<InMemoryKey<K>, InMemoryValue<K, V>> createMerkleMap(
             String label) {
         final var map = new MerkleMap<InMemoryKey<K>, InMemoryValue<K, V>>();
         map.setLabel(label);
         return map;
     }
 
-    /** Creates a new arbitrary virtual map with the given label, storageDir, and metadata */
+    /**
+     * Creates a new arbitrary virtual map with the given label, storageDir, and metadata
+     */
     @SuppressWarnings("unchecked")
     protected VirtualMap<OnDiskKey<String>, OnDiskValue<String>> createVirtualMap(
             String label, StateMetadata<String, String> md) {
-        final var keySerializer = new OnDiskKeySerializer<>(md);
-        final var builder = new JasperDbBuilder<OnDiskKey<String>, OnDiskValue<String>>()
-                .hashesRamToDiskThreshold(0)
-                .maxNumOfKeys(100)
-                .preferDiskBasedIndexes(true)
-                .keySerializer(keySerializer)
-                .virtualLeafRecordSerializer(new VirtualLeafRecordSerializer<>(
+        final MerkleDbTableConfig<OnDiskKey<String>, OnDiskValue<String>> tableConfig = new MerkleDbTableConfig<>(
+                        (short) 1,
+                        DigestType.SHA_384,
                         (short) 1,
-                        DataFileCommon.VARIABLE_DATA_SIZE,
-                        keySerializer,
+                        new OnDiskKeySerializer<>(md),
                         (short) 1,
-                        DataFileCommon.VARIABLE_DATA_SIZE,
-                        new OnDiskValueSerializer<>(md),
-                        false));
-        return new VirtualMap<>(label, builder);
+                        new OnDiskValueSerializer<>(md))
+                .hashesRamToDiskThreshold(0)
+                .maxNumberOfKeys(100)
+                .preferDiskIndices(true);
+
+        try {
+            final VirtualDataSourceBuilder<OnDiskKey<String>, OnDiskValue<String>> dsBuilder =
+                    new MerkleDbDataSourceBuilder<>(
+                            TemporaryFileBuilder.buildTemporaryDirectory("merkledb"), tableConfig);
+            return new VirtualMap<>(label, dsBuilder);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
If we're going to effectively hide an exception like this, at least add a diagnostic statement to publish that we're doing so.